packages/components/src/local/marker-form/index.tsx

```diff
@@ -71,6 +71,10 @@ export const MarkerForm: React.FC<PropTypes> = ({ formData, updateMarker, closeF
     updateMarker(UPDATE_MARKER, payload)
   }
 
+  const deleteMarker = (): void => {
+    console.log('handler to delete marker')
+  }
```

Comment:
I need a delete confirmation dialog to popup when `deleteMarker` is called.  If the user cancels, then no change. If the user confirms, then the code should call a `deleteMarker` callback.  Yes - I have some naming trouble here ;-). Let's rename this handler method as `deleteMarkerHandler`.

Reply:
I will take care of it @IanMayo 👍